app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx

```diff
@@ -213,7 +215,8 @@ AdminSidebarComponent.defaultProps = {
 
 export const AdminSidebar = track()(
   connect((state) => ({
-    activeProject: activeProjectSelector(state),
+    organizationSlug: projectOrganizationSlugSelector(state),
+    projectKey: activeProjectKeySelector(state),
```

Comment:
In this case the `projectKey` may belongs to a differ organization that `organizationSlug`.

Reply:
Done.